bin/get_metadata.py

```diff
@@ -79,27 +82,38 @@ def find_flowcell_type_novaseqx(self):
             return None
         return {"Flowcell type": flowcell_type}
 
-    def read_stats_json(self, bcl2fastq_outdir):
-        stats_json_path = os.path.join(
-            self.runfolder, bcl2fastq_outdir, "Stats/Stats.json"
-        )
+    def read_stats_json(self, demultiplexer_outdir, demultiplexer):
+        stats_path = "Reports" if demultiplexer == "bclconvert" else "Stats/Stats.json"
+        stats_json_path = os.path.join(self.runfolder, demultiplexer_outdir, stats_path)
         if os.path.exists(stats_json_path):
-            with open(stats_json_path) as f:
-                return json.load(f)
+            if demultiplexer == "bclconvert":
+                # Bclconvert produces multiple statistical output files
+                files = glob.glob(stats_json_path + "/*.csv")
+                bclconvert_data = {}
+                for file in files:
+                    with open(file) as csvfile:
+                        reader = csv.reader(csvfile)
+                        file_name = re.sub(r".*/|\.csv", "", file)
+                        bclconvert_data[file_name] = [row for row in reader]
+                return bclconvert_data
+            else:
+                with open(stats_json_path) as f:
+                    return json.load(f)
         else:
             return None
 
-    def get_bcl2fastq_version(self, runfolder):
-        with open(os.path.join(runfolder, "bcl2fastq_version")) as f:
-            bcl2fastq_str = f.read()
-        return bcl2fastq_str.split("v")[1].strip()
+    def get_demultiplexer_version(self, runfolder):
+        with open(os.path.join(runfolder, f"{self.demultiplexer}_version")) as f:
+            demultiplexer_str = f.read()
+        return demultiplexer_str.split("v")[1].strip()
 
     def get_software_version(self, runfolder):
-        with open(
-            Path(runfolder)
-            / "pipeline_info"
-            / "nf_core_pipeline_software_mqc_versions.yml"
-        ) as f:
+        pipeline_info_filename = (
+            "nf_core_pipeline_software_mqc_versions.yml"
```

Comment:
The name of this file has nothing to do with the demultiplexer used, only the demultiplex pipeline version. I think newer versions of the pipeline outputs `nf_core_demultiplex_software_mqc_versions.yml`. Perhaps we could use `glob` to find any file that matches `*software_mqc_versions.yml` ?